CONTRIBUTING.md

```diff
@@ -1,97 +1,31 @@
-example's directory, run `flutter packages get` to make sure its dependencies have been
-downloaded, and use `flutter run`. Make sure you have a device connected over
-USB and debugging enabled on that device. For example:
+Flutter packages follow [Flutter's style
+guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo), with the
+exception that code is formatted with `dart format`.
 
-- `cd packages/palette_generator/example`
-- `flutter packages get`
-- `flutter run`
+### Releasing
 
-## Contributing code
-
-We gladly accept contributions via GitHub pull requests.
-
-Please peruse our
-[style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) and
-[design principles](https://flutter.io/design-principles/) before
-working on anything non-trivial. These guidelines are intended to
-keep the code consistent and avoid common pitfalls.
-
-To start working on a patch:
-
-- `git fetch upstream`
-- `git checkout upstream/master -b <name_of_your_branch>`
-- Hack away.
-- Verify changes with [flutter_plugin_tools](https://pub.dartlang.org/packages/flutter_plugin_tools)
-
-```shell
-pub global activate flutter_plugin_tools
-pub global run flutter_plugin_tools format --plugins package_name
-pub global run flutter_plugin_tools analyze --plugins package_name
-pub global run flutter_plugin_tools test --plugins package_name
-```
-
-_If `pub` is not available, use `flutter pub` instead._
-
-- Check that the package can be published (but don't publish it until it has landed!):
-
-```shell
-cd packages/package_name; pub publish --dry-run
-```
-
-- `git commit -am "<your informative commit message>"`
-- `git push origin <name_of_your_branch>`
-
-To send us a pull request:
-
-- `git pull-request` (if you are using [Hub](http://github.com/github/hub/)) or
-  go to `https://github.com/flutter/packages` and click the
-  "Compare & pull request" button
-
-Please make sure all your checkins have detailed commit messages explaining the patch.
-
-Once you've gotten an LGTM from a project maintainer and once your PR has received
-the green light from all our automated testing (Travis, AppVeyor, etc), submit your
-changes to the `master` branch using one of the following methods:
-
-- Wait for one of the project maintainers to submit it for you.
-- Click the green "Merge pull request" button on the GitHub UI of your pull
-  request (requires commit access).
-
-You must complete the [Contributor License Agreement](https://cla.developers.google.com/clas).
-You can do this online, and it only takes a minute.
-If you've never submitted code before, you must add your (or your
-organization's) name and contact info to the [AUTHORS](AUTHORS) file.
-
-We grant commit access to people who have gained our trust and demonstrated
-a commitment to Flutter.
+If you are a team member landing a PR, or just want to know what the release
+process is for packages changes, see [the release
```

Comment:
```suggestion
process is for `packages` repository changes, see [the release
```
(or maybe "changes to this repository")

Reply:
Oops, it was just supposed to say "for package changes".